public/stac/HLS-L30.stac.json

```diff
@@ -0,0 +1,624 @@
+    },
+    "SAA": {
+      "raster:bands": [
+        {
+          "data_type": "uint16",
+          "nodata": 40000,
+          "unit": "degree",
+          "scale": 0.01,
+          "spatial_resolution": 30
+        }
+      ],
+      "title": "Sun Azimuth Angle (SAA)",
+      "roles": [
+        "data"
+      ],
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    },
+    "SZA": {
+      "raster:bands": [
+        {
+          "data_type": "uint16",
+          "nodata": 40000,
+          "unit": "degree",
+          "scale": 0.01,
+          "spatial_resolution": 30
+        }
+      ],
+      "title": "Sun Zenith Angle (SZA)",
+      "roles": [
+        "data"
+      ],
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    },
+    "VAA": {
+      "raster:bands": [
+        {
+          "data_type": "uint16",
+          "nodata": 40000,
+          "unit": "degree",
+          "scale": 0.01,
+          "spatial_resolution": 30
+        }
+      ],
+      "title": "View Azimuth Angle (VAA)",
+      "roles": [
+        "data"
+      ],
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    },
+    "VZA": {
+      "raster:bands": [
+        {
+          "data_type": "uint16",
+          "nodata": 40000,
+          "unit": "degree",
+          "scale": 0.01,
+          "spatial_resolution": 30
+        }
+      ],
+      "title": "View Zenith Angle (VZA)",
+      "roles": [
+        "data"
+      ],
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    }
+  },
+  "sci:publications": [
+    {
+      "doi": "10.1016/j.rse.2018.09.002",
+      "citation": "Claverie, M., Ju, J., Masek, J. G., Dungan, J. L., Vermote, E. F., Roger, J.-C., Skakun, S. V., & Justice, C. (2018). The Harmonized Landsat and Sentinel-2 surface reflectance data set. Remote Sensing of Environment, 219, 145-161."
+    }
+  ],
+  "title": "Harmonized Landsat Sentinel-2 L30",
+  "extent": {
+    "spatial": {
+      "bbox":[[-180,-90,180,90]]
+    },
+    "temporal": {
+      "interval":[["2013-04-11T00:00:00.000Z",null]]
+    }
+  },
+  "license": "proprietary",
+  "keywords": [
+    "NASA",
+    "USGS",
+    "Landsat",
+    "ESA",
+    "Copernicus",
+    "Sentinel",
+    "Satellite",
+    "Imagery",
+    "Global",
+    "Reflectance",
+    "Temperature"
+  ],
+  "providers": [
+    {
+      "name": "NASA LP DAAC at the USGS EROS Center",
+      "roles": [
+        "producer",
+        "licensor",
+        "processor"
+      ],
+      "url": "https://lpdaac.usgs.gov/"
+    }
+  ],
+
+  "summaries": {
+    "instruments": [
+      "oli",
+      "tiirs",
+      "msi"
+    ],
+    "platform": [
+      "landsat-8",
+      "landsat-9"
+    ],
+    "sci:doi": [
+      "10.5067/HLS/HLSL30.002"
+    ]
+  },
+  "renders": {
+    "truecolor": {
+      "title": "True Color",
+      "assets": [
+        "B04",
+        "B03",
+        "B02"
+      ],
```

Comment:
`assets` -> `bands` 
https://github.com/developmentseed/titiler-cmr/pull/19/files#diff-44ba68e73bc59efa3bed7c22f2a7a521883f385adcacc5e0f190194a85fff71fR471-R477

Reply:
we can also translate `assets` to `bands` in the client application (or change `bands` to `assets` in titiler-cmr)

Reply:
Does this mean we will be changing the titiler-cmr by replacing "bands" with "assets"?

I also noticed that the renders extension makes note that when needing to specify bands in an asset, a [virtual asset](https://github.com/stac-extensions/virtual-assets) should be used: https://github.com/stac-extensions/render?tab=readme-ov-file#assets-reference. Should we do that here?

Reply:
👍🏽